lib/load_disposable_domain.rb

```diff
@@ -0,0 +1,7 @@
+class LoadDisposableDomain
+  def self.load_disposable_domains(url)
+    Faraday.get(url).body.each_line do |line|
```

Comment:
If possible, I think we'll want to take the same approach as the pwned_passwords and have the file available without downloading.